azure/durable_functions/orchestrator.py

```diff
@@ -100,13 +100,22 @@ def handle(self, context: DurableOrchestrationContext):
                     actions=self.durable_context.actions,
                     custom_status=self.durable_context.custom_status)
             except Exception as e:
+                exception_str = str(e)
```

Comment:
to clarify, does this only grab the exception message, not the stack trace, right?

Reply:
You're correct, the `toString` / `str()` of an Exception gets its error message.